### PR TITLE
Fix seed for DC

### PR DESCRIPTION
**Pull request by shreyajain17**

### CODE REVIEW

File: src/ml/neural_net/tf_compute_context.hpp [hunk 1]
```diff
@@ -46,8 +46,8 @@ EXPORT class tf_compute_context: public compute_context {
 
   std::unique_ptr<model_backend> create_drawing_classifier(
       /* TODO: const float_array_map& config if needed */
```
Comment (shantanuchhabra) [inline src/ml/neural_net/tf_compute_context.hpp]: Let's remove the TODO

File: src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py [hunk 2]
```diff
@@ -27,6 +27,12 @@ def __init__(self, net_params, batch_size, num_classes):
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
 
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
+        else:
+            seed=None
```
Comment (shantanuchhabra) [inline src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py]: Let's follow PEP8 and add spaces around the assignment.

"Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not)." from https://www.python.org/dev/peps/pep-0008/

Comment (shreyajain17) [reply src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py]: I think I can remove the else completely. It is not needed to assign `seed` to None.

File: src/ml/neural_net/tf_compute_context.hpp (hunk 1 above)
Comment (shreyajain17) [reply src/ml/neural_net/tf_compute_context.hpp]: Okay

File: src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py [hunk 3]
```diff
@@ -27,6 +27,12 @@ def __init__(self, net_params, batch_size, num_classes):
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
 
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
```
Comment (nickjong) [inline src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py]: What is the random seed used for in the DC backend?

**Comment by shreyajain17** [comment]
> Can you add an unit test for this?

Done.

File: src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py (hunk 3 above)
Comment (shreyajain17) [reply src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py]: Didn't need this. Removed this and config everywhere.

**Pull request merged**